apps/erlangbridge/src/lsp_handlers.erl

```diff
@@ -291,14 +291,17 @@ severity(_) -> 1.
 
 auto_complete(File, Line, Text) ->
     RegexList = [
-        {"[^a-zA-Z0-9_@]([a-z][a-zA-Z0-9_@]*):((?:[a-z][a-zA-Z0-9_@]*)?)$", module_function},
-        {"#((?:[a-z][a-zA-Z0-9_@]*)?)$", record},
-        {"#([a-z][a-zA-Z0-9_@]*)\.((?:[a-z][a-zA-Z0-9_@]*)?)$", field},
-        {"[^a-zA-Z0-9_@]([A-Z][a-zA-Z0-9_@]*)$", variable},
-        {"^-([a-z]*)$", attribute},
-        {"([a-z][a-zA-Z0-9_@]*)$", atom}
+        {"[^a-zA-Z0-9_@](case)[^a-zA-Z0-9_@].*of\r?$", case_of},
```

Comment:
I've tested 'case of' autocompletion cancelation, and it does not work for me in most cases. It triggers on 'o' letter most of the time. So i suggest to modify regexp as follows:
```suggestion
        {"[^a-zA-Z0-9_@](case)[^a-zA-Z0-9_@].*\sof?\r?$", case_of},
```
It will cause more false positives but i think we can deal with it.

Reply:
Agreed, thank you!